public/app/plugins/panel/graph/series_overrides_ctrl.ts

```diff
@@ -147,6 +147,7 @@ export function SeriesOverridesCtrl($scope, $element, popoverSrv) {
   $scope.addOverrideOption('Points', 'points', [true, false]);
   $scope.addOverrideOption('Points Radius', 'pointradius', [1, 2, 3, 4, 5]);
   $scope.addOverrideOption('Stack', 'stack', [true, false, 'A', 'B', 'C', 'D']);
+  $scope.addOverrideOption('Percent', 'stackpercent', [true, false]);
```

Comment:
Does this option make more sense under "Stack" section?

Reply:
hello @torkelo,
I'm not sure to understand the question. Can you elaborate ?

Reply:
You placed it under a new percent category, I think under the existing stack category makes more sense

Reply:
Ok, I see and that make sense. But I don't  know how it can be done for overrides.
Do you mean by adding a specific value in the 'Stack' list ? If so, I fear we'll miss some use cases.
But it's fine for me if setting the 'Stack' value to false also disables the percent option. It won't make the set of options more complex and it ensures the options are reset to something stable. Let met know, I can provide another PR for that implementation.

Reply:
Sorry , ignore that comment, was confused about how override menu works